v23.1/striim.md

```diff
@@ -0,0 +1,105 @@
+
+### Initial data load
+
+To perform the initial load, create a Striim application and configure the source database using one of the **Initial Load** sources. Configure CockroachDB as a **PostgreSQL** target. For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster](connect-to-the-database.html). Do the following before deploying the application:
+
+- Specify the **Connection URL** in [JDBC format](connect-to-the-database.html?filters=java&#step-5-connect-to-the-cluster) while appending the `reWriteBatchedInserts=true` property, and without specifying the username. For example:
+
+	~~~
+	jdbc:postgresql://{host}:{port}/{database}?password={password}&sslmode=verify-full&reWriteBatchedInserts=true
+	~~~
+
+- After creating the Target, [export the App](https://www.striim.com/docs/platform/en/hands-on-quick-tour.html#UUID-a846e232-87e4-d88b-eb77-fa80691bbdf7) and add the field `_h_ConnectionRetryCode: '40001'` to the TQL file. For example:
+
+	~~~
+	CREATE OR REPLACE TARGET cockroach USING Global.DatabaseWriter ( 
+	  ConnectionRetryPolicy: 'retryInterval=30, maxRetries=3', 
+	  ParallelThreads: '5', 
+	  CheckPointTable: 'CHKPOINT', 
+	  BatchPolicy: 'EventCount:128,Interval:60', 
+	  Password_encrypted: 'true', 
+	  StatementCacheSize: '4', 
+	  CDDLAction: 'Process', 
+	  Password: 'xxxxx', 
+	  Tables: 'public.test_table_large,public.test_table_large', 
+	  CommitPolicy: 'EventCount:128,Interval:60', 
+	  DatabaseProviderType: 'Postgres', 
+	  PreserveSourceTransactionBoundary: 'false', 
+	  ConnectionURL: 'jdbc:postgresql://{host}:{port}/{database}?password={password}&sslmode=verify-full&reWriteBatchedInserts=true', 
+	  Username: 'root', 
+	  _h_ConnectionRetryCode: '40001', 
+	  adapterName: 'DatabaseWriter' ) 
+	~~~
+
+	Then [import the modified TQL file](https://www.striim.com/docs/platform/en/creating-apps-by-importing-tql.html) to create a new App.
+
+{{site.data.alerts.callout_info}}
+To minimize downtime for your migration, configure a separate [continuous replication](#continuous-replication) application before you deploy the initial load application. Once the initial load is complete, deploy the continuous replication application.
+{{site.data.alerts.end}}
+
+Deploying this application performs the initial load of data to CockroachDB. Remember that you should have already [created the schema objects](#before-you-begin) on CockroachDB.
+
+### Continuous replication
+
+To perform continuous replication of ongoing changes, create another Striim application and configure the source database using one of the **CDC** sources. Configure CockroachDB as a **PostgreSQL** target. For information about where to find the CockroachDB connection parameters, see [Connect to a CockroachDB Cluster](connect-to-the-database.html). Do the following before deploying the application:
```

Comment:
do we want to make mention to use the same settings as above for cdc in terms of setting crdb as a target?